src/resources/sprite_resource.rs

```diff
@@ -0,0 +1,58 @@
+
+use amethyst::prelude::*;
+use amethyst::renderer::{ImageFormat, SpriteSheet, SpriteSheetFormat, SpriteRender, Texture};
+use amethyst::assets::{Handle, Loader, AssetStorage};
+
+#[derive(Clone)]
+pub struct SpriteResource {
+    pub sheet: Handle<SpriteSheet>
+}
+
+impl SpriteResource {
+    pub fn sprite_render_for_player(&self) -> SpriteRender {
+        self.sprite_render_at_index(3)
+    }
+
+    pub fn sprite_render_for_shooting_player(&self) -> SpriteRender {
```

Comment:
Later it might be better to use constants on these indices so we can avoid to end up with a tons of `sprite_render_for_something` functions

Reply:
It is just a comment here, not has to be resolved now